benchmarks/dbscan/dbscan.cpp

```diff
@@ -71,7 +71,8 @@ int main(int argc, char *argv[])
 
   std::cout << "ArborX version    : " << ArborX::version() << std::endl;
   std::cout << "ArborX hash       : " << ArborX::gitCommitHash() << std::endl;
-  std::cout << "Kokkos version    : " << KokkosExt::version() << std::endl;
+  std::cout << "Kokkos version    : " << ArborX::KokkosExt::version()
+            << std::endl;
```

Comment:
@dalg24 I know you are going to hate this. Lets just not worry about it in this patch. I can do a patch later that replaces all `std::endl` with `\n` in the examples and benchmarks.